### PR TITLE
Added parse_tags function in util.py

### PR DESCRIPTION
**Pull request by lipis**
Good to have and to be soon used with _magic_ [:tophat:](http://magic.gae-init.appspot.com)

### CODE REVIEW

File: main/util.py [hunk 1]
```diff
@@ -188,6 +188,12 @@ def update_query_argument(name, value=None, ignore='cursor', is_list=False):
   return '%s%s' % (flask.request.path, '?%s' % query if query else '')
 
 
+def parse_tags(tags, separator=None):
```
Comment (gmist) [inline main/util.py]:
``` python
def parse_tags(tags, separator=config.TAG_SEPARATOR):
...
    tags = str(tags.strip()).split(separator)
```

Comment (lipis) [inline main/util.py]: I tried that.. but I was getting an error.. and gave up :|

Comment (gmist) [inline main/util.py]: Hm interesting... What type of error?

Comment (lipis) [inline main/util.py]:
```shell
ERROR    2015-05-08 17:18:41,636 wsgi.py:263] 
Traceback (most recent call last):
  File "/Users/lipis/google-cloud-sdk/platform/google_appengine/google/appengine/runtime/wsgi.py", line 240, in Handle
    handler = _config_handle.add_wsgi_middleware(self._LoadHandler())
  File "/Users/lipis/google-cloud-sdk/platform/google_appengine/google/appengine/runtime/wsgi.py", line 299, in _LoadHandler
    handler, path, err = LoadObject(self._handler)
  File "/Users/lipis/google-cloud-sdk/platform/google_appengine/google/appengine/runtime/wsgi.py", line 85, in LoadObject
    obj = __import__(path[0])
  File "/Users/lipis/work/lipis/gae-init/git/init/main/main.py", line 5, in <module>
    import config
  File "/Users/lipis/work/lipis/gae-init/git/init/main/config.py", line 28, in <module>
    import model
  File "/Users/lipis/work/lipis/gae-init/git/init/main/model/__init__.py", line 3, in <module>
    from .base import Base
  File "/Users/lipis/work/lipis/gae-init/git/init/main/model/base.py", line 9, in <module>
    import util
  File "/Users/lipis/work/lipis/gae-init/git/init/main/util.py", line 191, in <module>
    def parse_tags(tags, separator=config.TAG_SEPARATOR):
AttributeError: 'module' object has no attribute 'TAG_SEPARATOR'
```

Comment (gmist) [inline main/util.py]: It seems like a conflict of import ```import config.py``` in ```model/base.py``` (line 5).

Comment (gmist) [inline main/util.py]: Or in ```util.py```
``` python
from __future__ import absolute_import
import config - OK
```
config.py
``` python
import util - Ok
```

util.py 
``` python
import config - Oops
```

Comment (lipis) [inline main/util.py]: Yes, there is quite an _import fest_ :)

in the good old days we didn't import `config` or `utils` in the [model.py](https://github.com/gae-init/gae-init/blob/0.14.2/main/model.py#L13)

Comment (lipis) [inline main/util.py]: So are we doing anything special here or..?

Comment (lipis) [inline main/util.py]: If anything we'll solve it later..

**Pull request merged**